source/Tool/GenerateAction.cpp

```diff
@@ -13,6 +13,8 @@
 #include "ConfigImpl.hpp"
 #include "CorpusImpl.hpp"
 #include "AST/AbsoluteCompilationDatabase.hpp"
+#include "Plugins.hpp"
```

Comment:
this is in the wrong order

Reply:
What's the right order?

Reply:
"this folder" comes before "other folder:"
``
#include "ConfigImpl.hpp"
#include "CorpusImpl.hpp"
#include "Plugins.hpp"
#include "AST/AbsoluteCompilationDatabase.hpp"
```